### PR TITLE
Added tasks 1541-1542-1544-1545-1546

### PR DESCRIPTION
**Pull request by ThanhNIT**

### CODE REVIEW

File: src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // #Medium #String #Recursion #2022_04_11_Time_0_ms_(100.00%)_Space_41.2_MB_(90.34%)
 
 public class Solution {
-    public char findKthBit(int n, int k) {
+    public char findKthBit(int k) {
```
Comment (javadev) [inline src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java]: Please add SuppressWarnings annotation for non used argument.

Comment (javadev) [reply src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java]: The code is java:S1172

**Comment by javadev** [comment]
👍

File: src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java (hunk 1 above)
Comment (ThanhNIT) [reply src/main/java/g1501_1600/s1545_find_kth_bit_in_nth_binary_string/Solution.java]: thanks

**Pull request merged**